### ISSUE TITLE
Peripheral location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11560]
Preferred term label:
Peripheral location

Synonyms

    Peripherally located

    Peripheral region

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the outer edges, margins, or periphery of a structure. In ophthalmology, it is frequently used to describe peripheral retinal degenerations, peripheral visual field loss, or peripheral corneal thinning. PMID:23111175, PMID:30686588

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Retinitis pigmentosa (OMIM:268000) (e.g., peripheral vision loss)

    Terrien marginal degeneration (Orphanet:98956)

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by daboe01** [comment]
dupe to HP:0030646
